ignite/services/network/queries.go

```diff
@@ -220,3 +222,41 @@ func (n Network) ChainReward(ctx context.Context, launchID uint64) (rewardtypes.
 	}
 	return res.RewardPool, nil
 }
+
+// StakingParams fetches the staking module params
+func (n Network) StakingParams(ctx context.Context) (stakingtypes.Params, error) {
+	res, err := n.stakingQuery.Params(ctx, &stakingtypes.QueryParamsRequest{})
+	if err != nil {
+		return stakingtypes.Params{}, err
+	}
+	return res.Params, nil
+}
+
+// IBCInfo Fetches the consensus state with the validator set,
+// the unbounding time, and the last block height from chain rewards.
+func (n Network) IBCInfo(
+	ctx context.Context,
+	launchID uint64,
+	height int64,
+) (networktypes.IBCInfo, int64, int64, error) {
+	chainReward, err := n.ChainReward(ctx, launchID)
+	if err == ErrObjectNotFound {
+		return networktypes.IBCInfo{}, 0, 0,
+			fmt.Errorf("chain %d don't have rewards to launch an incentivized network", launchID)
+	} else if err != nil {
+		return networktypes.IBCInfo{}, 0, 0, err
+	}
+
+	ibcInfo, err := NodeInfo(ctx, n.cosmos, height)
+	if err != nil {
+		return networktypes.IBCInfo{}, 0, 0, err
+	}
+
+	stakingParams, err := n.StakingParams(ctx)
+	if err != nil {
+		return networktypes.IBCInfo{}, 0, 0, err
+	}
+	unboundingTime := int64(stakingParams.UnbondingTime.Seconds())
```

Comment:
```suggestion
	unboundingTime := uint64(stakingParams.UnbondingTime.Seconds())
```

Reply:
why use `uint64`? the [`unboundingTime`](https://github.com/tendermint/spn/blob/6df911993f135020c60db766db5a9c44baa6086f/x/monitoringc/types/msg_create_client.go#L19) is `int64`